outputs/core_pools.json

```diff
@@ -41,7 +42,6 @@
   },
   "arbitrum": {
     "0x125bc5a031b2db6733bfa35d914ffa428095978b000200000000000000000514": "ECLP-AUSDC-AUSDT",
-    "0x260dbd54d87a10a0fc9d08622ebc969a3bf4e6bb000200000000000000000536": "jitoSOL/wstETH",
```

Comment:
What is the limit.  There's 172k in this pool and 172k in fees.  It's not an issue, but I thought we had set this to 100k.